scripts/Client.js

```diff
@@ -2,13 +2,14 @@
  * current row and final row of the specified name globally know
  */
 var CURR_ROW = 0;
+var START_ROW = 0;
 var END_ROW = 0;
 
-
 /**
  * ID of spreadsheet with sample data.
  */
-var SPREADSHEET_ID = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx';
+//var SPREADSHEET_ID = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx';
+var SPREADSHEET_ID = '19kcAFdRSmJM9_LG-Tk5uqM3vzY3y700Du4Rt72iBie0';
```

Comment:
can you please delete this line

Reply:
It's been deleted in the commit after this one!